src/renderer/components/dialogs/Settings-Profile.tsx

```diff
@@ -167,14 +169,16 @@ export function ProfileImageSelector({
         >
           {tx('profile_image_select')}
         </button>
-        <button
-          aria-label={tx('profile_image_delete')}
-          onClick={onClickRemovePicture}
-          className={'delta-button-round'}
-          disabled={!profilePicture}
-        >
-          {tx('profile_image_delete')}
-        </button>
+        {!hideDeleteButton ? (
+          <button
+            aria-label={tx('profile_image_delete')}
+            onClick={onClickRemovePicture}
+            className={'delta-button-round'}
+            disabled={!profilePicture}
+          >
+            {tx('profile_image_delete')}
+          </button>
+        ) : null}
```

Comment:
users might reconsider their choice so I would still offer the option to delete the profile image again. sure the two buttons are not that pretty, but I think improving that is something for later.